IGRGE1/Calculator/ViewModel/MainViewModel.cs

```diff
@@ -0,0 +1,394 @@
+
+namespace Calculator
+{
+
+
+    public class MainViewModel : ViewModelBase, INotifyPropertyChanged
+    {
+        private string totalText = "";
+        private string subTotalText = "";
+        private List<string> lines = new();
+        public readonly record struct MinMax(double Minimum, double Maximum);
+        double resultHelper = 0;
+        public string TotalText { get => totalText; set => SetProperty(ref totalText, value); }
+        public string SubTotalText { get => subTotalText; set => SetProperty(ref subTotalText, value); }
+        public ICommand NotZeroNumber_Command { get; private set; }
+        public ICommand ZeroNumber_Command { get; private set; }
+        public ICommand ExportToJSONMinMax_Command { get; private set; }
+        public ICommand FindMin_Command { get; private set; }
+        public ICommand FindMax_Command { get; private set; }
+        public ICommand Divide_Command { get; private set; }
+        public ICommand Multiplication_Command { get; private set; }
+        public ICommand ClearTotalTextBox { get; private set; }
+        public ICommand CommaSeparator_Command { get; private set; }
+        public ICommand WriteToMemoryAdd_Command { get; private set; }
+        public ICommand Addition_Command { get; private set; }
+        public ICommand Substraction_Command { get; private set; }
+        public ICommand Equation_Command { get; private set; }
+        public event PropertyChangedEventHandler PropertyChanged;
+        protected bool SetProperty<T>(ref T field, T newValue, [CallerMemberName] string propertyName = null)
+        {
+            if (!(object.Equals(field, newValue)))
+            {
+                field = (newValue);
+                PropertyChanged?.Invoke(this, new PropertyChangedEventArgs(propertyName));
+                return true;
+            }
+
+            return false;
+        }
+
+        public MainViewModel()
+        {
+            ClearTotalTextBox = new RelayCommand(ClearTextBox);
+            FindMin_Command = new RelayCommand(FindMinAsync);
+            ZeroNumber_Command = new RelayCommand(ZeroNumber_Click);
+            NotZeroNumber_Command = new RelayCommand<string>(NotZeroNumber_Click);
+            FindMax_Command = new RelayCommand(FindMaxAsync);
+            CommaSeparator_Command = new RelayCommand(CommaSeparator);
+            WriteToMemoryAdd_Command = new RelayCommand(WriteToMemoryAddAsync);
+            ExportToJSONMinMax_Command = new RelayCommand(ExportToJSONMinMaxAsync);
+            Addition_Command = new RelayCommand(Addition);
+            Substraction_Command = new RelayCommand(Substraction);
+            Multiplication_Command = new RelayCommand(Multiplication);
+            Divide_Command = new RelayCommand(Divide);
+            Equation_Command = new RelayCommand(Equation);
+        }
+
+        private void Divide()
+        {
+            if (SubTotalText.Equals("") && !TotalText.Equals("") && !TotalText.Equals("-"))
```

Comment:
Equals() hívás String esetén annyira Java beütés :) Itt van == operátor, ami String esetén is működik.